src/lib/tasks/server/load_server_file_task.cpp

```diff
@@ -8,7 +8,7 @@ namespace opossum {
 
 void LoadServerFileTask::_on_execute() {
   try {
-    const auto table = load_table(_file_name, Chunk::MAX_SIZE);
+    const auto table = load_table(_file_name, Chunk::DEFAULT_SIZE);
```

Comment:
We could get rid of explicitly mentioning `Chunk::DEFAULT_SIZE` here. It does not really matter, but in sql_benchmark.cpp you removed this parameter.

Reply:
Yepp, missed than one, thanks.